app/src/main/java/de/tum/in/tumcampusapp/component/tumui/grades/GradesFragment.kt

```diff
@@ -184,6 +191,12 @@ class GradesFragment : FragmentForAccessingTumOnline<ExamList>(
                     )
             )
 
+            // TODO exchange deprecated function
```

Comment:
Whats this about? is getColor deprecated?

Reply:
Yes, since more than 5 years... Haha
Well, I'm not completely sure why I didn't use another appropriate method, but I think this was because of any api violences that I didn't like to address here.